packages/react-native-sdk/src/components/LobbyView.tsx

```diff
@@ -47,8 +47,9 @@ export const LobbyView = ({ joinButton }: LobbyViewType) => {
     isCameraOnFrontFacingMode,
   } = useMediaStreamManagement();
   const isVideoAvailable = !!localVideoStream && initialVideoEnabled;
-  const count = useParticipantCount();
```

Comment:
good fix,..

this is basically sfu vs coordinator

in lobby we should not use the sfu count ;). well spotted